deploy/crds.yaml

```diff
@@ -612,6 +612,10 @@ spec:
                 items:
                   description: TransportServerUpstream defines an upstream.
                   properties:
+                    backup:
```

Comment:
Use `make update-crds` command to update this file if not already 👍🏼

Reply:
Updated using `make update-crds`